core/invoker/src/main/scala/whisk/core/containerpool/ContainerPool.scala

```diff
@@ -78,27 +76,56 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
     }
   }
 
+  def logContainerStart(r: Run, containerState: String): Unit = {
+    val namespaceName = r.msg.user.namespace.name
+    val actionName = r.action.name.name
+    val activationId = r.msg.activationId.toString
+
+    logging.info(
+      this,
+      s"containerStart containerState: $containerState action: $actionName namespace: $namespaceName activationId: $activationId")(
+      r.msg.transid)
+    MetricEmitter.emitCounterMetric(LoggingMarkers.INVOKER_CONTAINER_START(actionName, namespaceName, containerState))
+  }
```

Comment:
Why not `transid.mark`?

Reply:
Code evolvement reasons. We added the logging out output late (and with it the transaction id) and did no realize that we now were able to merge the two things using txid.mark. Corrected.